src/ImpactX.cpp

```diff
@@ -172,6 +174,14 @@ namespace impactx {
             amrex::Print() << " Space Charge effects: " << space_charge << "\n";
         }
 
+        bool csr = false;
+        int csr_bins = 100;
```

Comment:
Not using the documented default from `parameters.rst`, will fix in Python-binding push:
```suggestion
        int csr_bins = 150;
```